cli/messages.go

```diff
@@ -315,7 +315,7 @@ Provide a [transfer_id] argument to list files in the specified transfer.`
 	MsgFileshareAcceptUsage       = "Accept an incoming file transfer. To download an entire transfer, specify the transfer ID. To download a single file, specify the transfer ID and the file ID."
 	MsgFileshareAcceptArgsUsage   = "<transfer_id> [file_id1] [file_id2...]"
 	MsgFileshareAcceptDescription = MsgFileshareAcceptUsage + "\n\nTo cancel a transfer in progress, press Ctrl+C"
-	MsgFileshareAcceptPathUsage   = "Specify download path (default: $XDG_DOWNLOAD_DIR or $HOME/Downloads)"
+	MsgFileshareAcceptPathUsage   = "Specify download path (default: $XDG_DOWNLOAD_DIR or $HOME/Downloads or $SNAP_USER_DATA/Downloads for snap)"
```

Comment:
not sure if this makes sense to add, if for some systems we cannot access $HOME/Downloads

Reply:
Changes were reverted.